CHANGELOG.md

```diff
@@ -29,6 +35,12 @@
     https://github.com/RevenueCat/purchases-flutter/pull/285
 - Updated JSON serialization using Freezed
     https://github.com/RevenueCat/purchases-flutter/pull/270
+- Breaking changes:
```

Comment:
Perhaps we can add a note for full transparency here, explaining that this was unintended and fixed in 3.9.4? 
Just so it doesn't look like we _intended_ to make breaking changes in a hotfix

Reply:
sure, good call

Reply:
fixed in this pr and in the gh release..